src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.security.SecureRandom;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new SecureRandom().nextInt(100));
```

Comment:
magic number. Lets use the Random class instead of SecureRandom and make it a constants to ColorSupplier and Random